sysbuild/Kconfig.netcore

```diff
@@ -53,4 +65,27 @@ config NETCORE_HCI_RPMSG_DOMAIN_APP
 	bool
 	default y if NETCORE_HCI_RPMSG_BOARD = "nrf5340dk_nrf5340_cpunet"
 
+config NETCORE_802154_RPMSG_NAME
+	string
+	default "802154_rpmsg"
+	depends on NETCORE_802154_RPMSG
+	help
+	  Image name of radio 802154 rpmsg.
+
+config NETCORE_802154_RPMSG_BOARD
```

Comment:
The current system (this) is unworkable and has no future, but requires boards v2 to be finished, reviewed and merged into zephyr so it can be used, for example any nRF5340-based board that isn't the nRF5340dk needs to have all these files manually updated.

Reply:
So you want to merge this PR in the current state and improve it later on?

Reply:
Boards V2 is a very long way off: https://github.com/zephyrproject-rtos/zephyr/pull/50305

Reply:
I see. Thanks for the clarification